core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -566,10 +801,19 @@ object ConsumerGroupCommand extends Logging {
         CommandLineUtils.checkRequiredArgs(parser, options, groupOpt)
       if (options.has(deleteOpt) && !options.has(groupOpt) && !options.has(topicOpt))
         CommandLineUtils.printUsageAndDie(parser, "Option %s either takes %s, %s, or both".format(deleteOpt, groupOpt, topicOpt))
+      if (options.has(resetOffsetsOpt))
```

Comment:
If we assert that at least one of `allResetOffsetScenarioOpts` is provided, then we don't need the weird case above where we reset to the current offsets.